DREAM3DReviewFilters/GenerateMaskFromSimpleShapes.cpp

```diff
@@ -0,0 +1,669 @@
+/*
+ * Your License or Copyright can go here
+ */
+
+#include "GenerateMaskFromSimpleShapes.h"
+
```

Comment:
```suggestion

 #include <array>
 
```
Added for std::array.